guillotina/auth/users.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, user_id="guillotina", groups=None, roles=None, permissions=No
         self.id = user_id
         self._groups = groups or []
         self._roles = roles or {}
+        self._roles["guillotina.Authenticated"] = Allow
```

Comment:
I don't think this is safe. `_roles` might be overridden. Adding the `guillotina.Authenticated` role as part of the auth process is on purpose.